packages/components/accordion/src/accordion-item.tsx

```diff
@@ -24,25 +24,65 @@ import { AccordionPanel } from "./accordion-panel"
 
 export interface AccordionItemOptions {
   children?:
-    | ((props: { isDisabled: boolean; isExpanded: boolean }) => ReactNode)
+    | ((props: {
+        /**
+         * @deprecated Use `disabled` instead.
+         */
+        isDisabled: boolean
+        /**
+         * @deprecated Use `expanded` instead.
+         */
+        isExpanded: boolean
+        disabled?: boolean
+        expanded?: boolean
+      }) => ReactNode)
     | ReactNode
+  /**
+   * If `true`, the accordion item will be disabled.
+   *
+   * @default false
+   */
+  disabled?: boolean
   /**
    * The accordion icon to use.
    */
   icon?:
-    | ((props: { isDisabled: boolean; isExpanded: boolean }) => ReactNode)
+    | ((props: {
+        /**
+         * @deprecated Use `disabled` instead.
+         */
+        isDisabled: boolean
+        /**
+         * @deprecated Use `expanded` instead.
+         */
+        isExpanded: boolean
+        disabled?: boolean
+        expanded?: boolean
+      }) => ReactNode)
     | ReactNode
   /**
    * If `true`, the accordion item will be disabled.
    *
    * @default false
+   * @deprecated Use `disabled` instead.
```

Comment:
Spacing one line apart.

Reply:
@cidkumagai 
I don't understand why spacing one line apart is necessary.
I intended to work on it the same way as the other components, but is there something different?

Rotate.tsx
![스크린샷 2024-11-16 10 49 13](https://github.com/user-attachments/assets/f896ab37-99d3-4eda-ade3-d053e4ea2df6)

Reply:
@kaehehehe 
I felt that I should follow the existing code since most of it includes line breaks.

Reply:
"I just adapted to the style of the person who started working first, but it seems that the style varies depending on the person who worked."

Reply:
Fixed.

Reply:
I checked myself, and it's not limited to `@deprecated`.